### PR TITLE
Fix description for Kafka and instance ids

### PR DESCRIPTION
**Pull request by indigo423**
### All Contributors

* [ ] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [ ] Have you made a comment in that issue which points back to this PR?
* [ ] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new or updated feature, is there documentation for the new behavior?
* [ ] If this is new code, are there unit and/or integration tests?
* [ ] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

Fix description for Kafka and the usage for instance ids. IMHO, this is nothing we need to track in the release notes so no JIRA issue :)

### CODE REVIEW

File: docs/modules/reference/pages/configuration/tuning-kafka.adoc [hunk 1]
```diff
@@ -12,8 +12,9 @@ Find available configuration parameters for Kafka here:
 
 == Multiple {page-component-title} instances
 
-Topics will be automatically created and are prefixed by default with `OpenNMS`. 
-If you want to use an Apache Kafka cluster with multiple {page-component-title} instances, customize the topic prefix by setting `org.opennms.core.ipc.rpc.kafka.group.id`, `org.opennms.core.ipc.sink.kafka.group.id`, and `org.opennms.instance.id` in {page-component-title} and its Minions to a string value that identifies your instance.
+Topics will be automatically created and are prefixed by default with OpenNMS.
+If you want to use an Apache Kafka cluster with multiple {page-component-title} instances, customize the topic prefix by setting `org.opennms.instance.id` in {page-component-title} and its Minions to a string value that identifies your instance.
+If you're using Sentinels, that must be used there as well.
```
Comment (dino2gnt) [inline docs/modules/reference/pages/configuration/tuning-kafka.adoc:17]:
```suggestion
If you want to use an Apache Kafka cluster with multiple {page-component-title} instances, customize the topic prefix by setting the `org.opennms.instance.id` system property on {page-component-title} and all its Minions and Sentinels to a string value that identifies your instance.
```

Comment (Bonrob2) [reply docs/modules/reference/pages/configuration/tuning-kafka.adoc:17]:
```suggestion
To use an Apache Kafka cluster with multiple {page-component-title} instances, customize the topic prefix by setting `org.opennms.instance.id` in {page-component-title} and its Minions to a string value that identifies your instance.
If you're using Sentinels, you must do the same there as well.
```

**Review by dino2gnt — APPROVED** [review]
LGTM

**Comment by indigo423** [comment]
> To use an Apache Kafka cluster ....

I've merged Dino's suggestion, is this still an issue?

**Comment by Bonrob2** [comment]
> > To use an Apache Kafka cluster ....
> 
> @Bonrob2 I've merged Dino's suggestion, is this still an issue?

It's not a big deal. Just a suggestion to make the sentence shorter. The merged version conveys the same meaning, so it's fine as is.

**Pull request merged**